src/pages/library/dataBrowser-utils.ts

```diff
@@ -193,16 +199,13 @@ interface DataCatalog {
 export const useDataCatalog = (): DataCatalog => {
   const signal = useCancellation();
   const [loading, setLoading] = useState(false);
-  const dataCatalog = useStore(dataCatalogStore) as Dataset[];
+  const dataCatalog = useStore(dataCatalogStore);
```

Comment:
Added the type to the store in libs/state so a cast is no longer necessary here.